src/lib/device/virtual_dev.hpp

```diff
@@ -99,10 +99,11 @@ class VirtualDev {
     blk_allocator_type_t m_allocator_type;
     chunk_selector_type_t m_chunk_selector_type;
     bool m_auto_recovery;
+    bool m_use_slab_in_blk_allocator;
```

Comment:
I think we should add a parameter `use_slab_allocator` inside `struct vdev_parameters` and let each service to set this field while calling `DeviceManager::create_vdev`, and in `populate_vdev_info` it needs to be persistent in vdev_info as well to survive recovery restore of the meta service vdev.